src/hb-ot-color-colr-table.hh

```diff
@@ -217,6 +217,7 @@ struct COLR
 				BaseGlyphRecord new_record;
```

Comment:
Alternative fix is to add `= {0}` initialization here.

Reply:
Done. Thanks for the suggestion :)

Reply:
Yep I suggested `{0}` instead of `{}` because I had a vague memory of the latter being a GNU extension. But you are correct that that's only in C. We're good. Thanks!